### ISSUE TITLE
python3: can't open file '/content/app.py': [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by che2ze** [jhj0517/Whisper-WebUI#115]
(python3: can't open file '/content/app.py': [Errno 2] No such file or directory)
 - OS: [windows]
 - 
![image](https://github.com/jhj0517/Whisper-WebUI/assets/164759760/2e70fddf-83ed-4d27-b14f-13cd188e1ddb)
Do I always have to reinstall it by pressing the 'Installation' button when I go in to use it again the next day?

**Comment by jhj0517** [comment]
Hi. This happens because you might have moved your current location during the installation
If you move to the Whisper-WebUI folder with `%cd Whisper-WebUI` or just reset and run the notebook again, it will be fine.

By the way, I discovered that the installation time increased significantly in Colab. So, I tried to fix this in #116.
The installation time has been reduced 10 minutes -> 1 minute in my experience.

And
>> Do I always have to reinstall it by pressing the 'Installation' button when I go in to use it again the next day?

If you're using Colab, yes. Since you "borrow" the session from Colab, the session resets everything after you're disconnected.
